accessing-betydb-with-arcmap.md

```diff
@@ -0,0 +1,174 @@
+# Accessing BetyDB from ArcMap
+Interested researchers can access BetyDB directly from ArcMap. In some
+cases direct access can simplify manipulation of the BetyDB data but
+this convenience must be weighed against a more complex setup, limits of
+ArcMap compatibility, and additional complexity of extracting data from
+a SQL database.
+
```

Comment:
Mention that - these instructions are intended for collaborators within the TERRA program. For users outside the TERRA program, access directly to the Postgres database requires setting up a clone. Instructions for using load.bety.sh should be copied from here: https://github.com/terraref/reference-data/issues/111